invenio_app_ils/ext.py

```diff
@@ -177,7 +177,10 @@ def init_app(self, app):
         app.extensions["invenio-app-ils"] = _InvenioAppIlsState(app)
         app.register_blueprint(
             Blueprint(
-                "invenio_app_ils_static", __name__, static_folder="static"
+                "invenio_app_ils",
+                __name__,
+                static_folder="static",
+                template_folder="templates",
```

Comment:
These probably should exist

Reply:
can you elaborate? It does exist, it contains the common `footer.html`